common/model/contributors.js

```diff
@@ -5,7 +5,7 @@ import type {HTMLString} from '../services/prismic/types';
 
 export type Organisation = {|
   name: string,
-  image: ?Picture,
+  image: Picture,
```

Comment:
is this really required?

Reply:
Yup - if we don't get one from Prismic we will placehold it.
Thinking about getting [Proxies](vernacular) in for this.